config/config.go

```diff
@@ -59,14 +59,12 @@ type AttrConfig struct {
 func NewAttrConfig(cfg config.Service) (*AttrConfig, error) {
 	attrCfg := &AttrConfig{}
 
-	_, err := config.TransformAttributeMapToStruct(attrCfg, cfg.Attributes)
-	if err != nil {
+	if _, err := config.TransformAttributeMapToStruct(attrCfg, cfg.Attributes); err != nil {
```

Comment:
[opt] I have avoided using this error-handling syntax in places like this because it makes the (already complicated) line much more complicated. Though, it might just be because I am new to reading go. Feel free to mark as resolved after you read this.

Reply:
This is idiomatic error handling for go.